packages/editor/src/components/properties/VariantNodeEditor.tsx

```diff
@@ -96,11 +96,12 @@ export const VariantNodeEditor: EditorComponentType = (props: { entity: Entity }
   const { t } = useTranslation()
   const entity = props.entity
   const variantComponent = useComponent(entity, VariantComponent)
-  const modelComponent = useComponent(entity, ModelComponent)
   const previewIndex = useHookstate(0)
 
   const setPreview = (index: number) => {
     previewIndex.set(index)
+    const modelComponent = useOptionalComponent(entity, ModelComponent)
```

Comment:
should be getOptionalMutableComponent instead of use

Reply:
Done!